src/stylesheets/component/_mainheader.scss

```diff
@@ -0,0 +1,85 @@
+
+.main-header{
+    width: 1043px;
```

Comment:
Why do you have to set the width height here?
Don't fix width and height in pixels that will make it difficult for you when you're responsive